lib/apiScenario/postmanCollectionRunnerClient.ts

```diff
@@ -783,6 +826,12 @@ try {
       });
     }
 
+    PostmanHelper.addEvent(pollerItem.events, "prerequest", [
+      `
+    console.log(pm.variables.get("x_polling_url"))
```

Comment:
we can add `verbose` option condition for this

Reply:
this is unused, will remove it